packages/react/error-boundary/src/ErrorBoundaryGroup.tsx

```diff
@@ -1,18 +1,11 @@
 /** @tossdocs-ignore */
-import {
-  ComponentPropsWithoutRef,
-  ComponentType,
-  createContext,
-  ReactNode,
-  useContext,
-  useEffect,
-  useMemo,
-  useRef,
-} from 'react';
+import { ComponentType, createContext, ReactNode, useContext, useEffect, useMemo, useRef } from 'react';
 import { useIsMounted, useKey } from './hooks';
+import { ComponentPropsWithoutChildren } from './types';
+import { isDevelopment } from './utils';
 
 export const ErrorBoundaryGroupContext = createContext({ resetKey: {}, reset: () => {} });
-if (process.env.NODE_ENV !== 'production') {
+if (isDevelopment) {
```

Comment:
I added isDevelopment to repetitive implementation of if current mode is development

Reply:
Actually I believe inlining `process.env.NODE_ENV` in the code is better, since we usually use [bundler's define feature](https://webpack.kr/plugins/define-plugin/) and [terser](https://github.com/terser/terser) to optimize this.

For example, when defining `process.env.NODE_ENV` to `'production'` this code becomes

```tsx
if ("production" !== 'production') {
  ErrorBoundaryGroupContext.displayName = 'ErrorBoundaryGroupContext';
}
```

which soon can be easily optimized by terser like this.

<img width="961" alt="image" src="https://user-images.githubusercontent.com/3102175/215373242-c28cd9fd-1b55-4c06-ba1b-d999c7ee9fca.png">

However when we make this as a imported variable I think the optimization will be harder.

For example, [React is also inlining the condition in its source code](https://github.com/facebook/react/blob/ce09ace9a23b61dfba53f5a83d52b377bc642bae/packages/react/npm/index.js#L3).

Reply:
Thanks for advise.

I rollbacked it in 58501e8